plugins/chain/serialize_state.cpp

```diff
@@ -290,6 +294,14 @@ void plugin::serialize_state(const bfs::path& output) {
         // custom_pack::_current_str_type = custom_pack::other;
         // STORE(proposal_index);                       // not supported
         // STORE(required_approval_index);              // not supported
+        auto reputation_flag = output;
+        reputation_flag += ".reputation";
+        if (db_.has_index<golos::plugins::follow::reputation_index>()) {
+            bfs::ofstream(reputation_flag);
+            serialize_table<golos::plugins::follow::reputation_index>(db_, out, _serialize_reputation_type, [](auto& h, auto& i){}, [](const auto& i){return true;});
```

Comment:
not understand idea with `reputation_flag`.
we can't write to the same serialized state file, because serialized state should be the same on every node

Reply:
agree. separate files are better for hashing. fixed